tests/code-splitting/subtests/dynamic-import-preloading/parcel/index.md

```diff
@@ -0,0 +1,5 @@
+---
+result: fail
+---
+
+Parcel does not fine grained control over code splitting which is required to for this test.
```

Comment:
I think it's possible to trigger this behavior in parcel.

Reply:
okay, I'll try again this weekend.

Reply:
@sokra I tried it. And it seems like the shared dependency `exclaim.js` is already into the parent module(index.js). 
According to parcel docs, 
```
If an asset is required in more than one bundle, it is hoisted up to the nearest common ancestor in 
the bundle tree so it is not included more than once.
```

Does that mean it passes the test? Even though it's not the ideal behavior.